src/protocols/secure_channel/SessionIDAllocator.h

```diff
@@ -46,7 +46,7 @@ class SessionIDAllocator
     static constexpr uint16_t kMaxSessionID       = UINT16_MAX;
     static constexpr uint16_t kUnsecuredSessionId = 0;
 
-    uint16_t mNextAvailable = 1;
+    static uint16_t mNextAvailable;
```

Comment:
Is this intentional? 
We may want a comment on why static is correct and desirable here.

Reply:
This was merged in a separate PR, so resolved.